File: SEM-3/Python/practical 4/program1.py
#1)
n1=int(input("Enter 1st number: "))
n2=int(input("Enter 2nd number: "))

print("Operation available: \nAddition : +\nSubtraction : -\nMultiplication : *\nDivision : /\nFloor Division : //\nExponential : **\nModulus : %")
op=input("Enter the Operation you want to perform: ")

if op=="+":
    print("Addition = ",n1+n2)
elif op=="-":
    print("Subtraction = ",n1-n2)
elif op=="*":
    print("Multiplition = ",n1*n2)
elif op=="/":
    print("Division = ",n1/n2)
elif op=="//":
    print("Floor Division = ",n1//n2)
elif op=="**":
    print("Exponential = ",n1**n2)
elif op=="%":
    print("Modulus = ",n1%n2)

File: SEM-3/dsa sem 3/Practical 2/program1.c
//1)
#include <stdio.h>

void main(){
    int n1,n2;
    int i;
    printf("Enter 1st number: ");
    scanf("%d",&n1);
    printf("Enter 2nd number: ");
    scanf("%d",&n2);
    printf("Operation you want to perform:\n1) Addition\n2) Subtraction\n3) Multiplication\n4) Division\n5) Modulus\n Enter the option(1,2,3,4,5): ");
    scanf("%d",&i);

    switch (i){
        case 1:
            printf("Addition = %d",n1+n2);
            break;
        case 2:
            printf("Subtraction = %d",n1-n2);
            break;
        case 3:
            printf("Multiplication = %d",n1*n2);
            break;
        case 4:
            printf("Division = %d",n1/n2);
            break;
        case 5:
            printf("Modulus = %d",n1%n2);
            break;
    }   
}